plenum/test/primary_selection/test_primary_selection_on_propagate_primary.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+
+from stp_core.common.log import getlogger
+
+from plenum.test.helper import sendReqsToNodesAndVerifySuffReplies
+
+from plenum.test.test_node import TestNode
+from plenum.test.view_change.helper import ensure_view_change_complete
+
+from plenum.test.node_catchup.conftest import nodeCreatedAfterSomeTxns, \
+    nodeSetWithNodeAddedAfterSomeTxns
+
+logger = getlogger()
+
+nodes_wth_bls = 0
```

Comment:
Did we set it intentionally?

Reply:
yes, it seemed necessary in the past, removed